### PR TITLE
Restore shortcuts

### PR DESCRIPTION
**Pull request by marcofernandezheras**
resetshortcuts

Added Link to faq, reset & clear all shortcuts buttons

### CODE REVIEW

**Comment by Daenyth** [comment]
What's the difference between clear and reset? As a user that's confusing. Does it unbind everything? When is that useful?

**Comment by marcofernandezheras** [comment]
Clear is to delete all shortcuts, reset is to put all shortcuts to default.
Clear is usefull if you want a fresh start

**Comment by Daenyth** [comment]
I see. That makes a little sense.

**Comment by ZeldaZach** [comment]
I like it

**Comment by tooomm** [comment]
`Reset all (shortcuts) to default` as label instead?

**Comment by ZeldaZach** [comment]
So this works fully as expected. I don't really see an issue with the translation of the buttons as @tooomm but I'm open to hearing your opinion, @marcofernandezheras.

This is ready to go whenever we get wording done properly :shipit:

File: cockatrice/src/sequenceEdit/shortcutstab.cpp [hunk 1]
```diff
@@ -16,3 +23,33 @@ ShortcutsTab::~ShortcutsTab()
...
+}
+
+void ShortcutsTab::refreshEdits()
+{
+    QList<SequenceEdit*> edits = this->findChildren<SequenceEdit*>();
+    for(int i= 0; i < edits.length(); ++i)
+    {
+        edits.at(i)->refreshShortcut();
+    }
+    QMessageBox::information(this,tr("Shortcuts reset"),tr("All shortcuts have been reset"));
```
Comment (tooomm) [inline cockatrice/src/sequenceEdit/shortcutstab.cpp]: "All shortcuts have been reset **to their defaults**"

File: cockatrice/src/sequenceEdit/shortcutstab.cpp [hunk 2]
```diff
@@ -16,3 +23,33 @@ ShortcutsTab::~ShortcutsTab()
...
+}
+
+void ShortcutsTab::afterClear()
+{
+    QList<SequenceEdit*> edits = this->findChildren<SequenceEdit*>();
+    for(int i= 0; i < edits.length(); ++i)
+    {
+        edits.at(i)->clear();
+    }
+    QMessageBox::information(this,tr("Shortcuts reset"),tr("All shortcuts have been cleared"));
```
Comment (tooomm) [inline cockatrice/src/sequenceEdit/shortcutstab.cpp]: We have 2 messages boxes here with the same name/title description, but they do two completely different things.

Other languages use several terms for this different actions.
That will bring some trouble for translators and makes the UI less simple.

**Comment by tooomm** [comment]
Is there some kind of confirmation dialog?

I don't like to miss click something and completely throw away my settings.
Since the settings window stores all changes immediatly (#1334) and doesn't wait until the `OK` button is clicked to close the settings dialog window AND save all changes, I like to see a confirmation dialog here for clearing/restoring shortcuts.

**Comment by Daenyth** [comment]
If we don't have an "undo" action we should have a confirmation. I
prefer having no confirmation but with "undo" available, but that's a
bit harder to write.

**Comment by tooomm** [comment]
I've barely seen a "undo" action in any application... ;P
A confirmation is not too embarrassing if you ask me and a nice tradeoff.

**Comment by Nightfirecat** [comment]
I'm in agreement that there should be either a confirmation for resetting/deleting all shortcuts or a simple method to undo it. My personal preference is for a confirmation dialog, since it's more clear and doesn't require any considerations for UI/UX for pointing a user to the "undo" functionality.

**Comment by marcofernandezheras** [comment]
Change button label to "Restore all default shortcuts"

**Comment by ZeldaZach** [comment]
Very nice job @marcofernandezheras 

I don't really care for the popup after the fact ("Your shortcuts have been cleared/reset to default!"). I find is unnecessary.

Otherwise, this is perfect!

**Pull request merged**